examples/guided.py

```diff
@@ -67,24 +84,31 @@ def ask_user_questions():
 					partition_mountpoints[partition] = None
 			except archinstall.UnknownFilesystemFormat as err:
 				archinstall.log(f" {partition} (Filesystem not supported)", fg='red')
-		
 
 		# We then ask what to do with the partitions.
 		if (option := archinstall.ask_for_disk_layout()) == 'abort':
-			archinstall.log(f"Safely aborting the installation. No changes to the disk or system has been made.")
+			archinstall.log(
+				f"Safely aborting the installation. No changes to the disk or system has been made."
+			)
 			exit(1)
 		elif option == 'keep-existing':
 			archinstall.arguments['harddrive'].keep_partitions = True
 
-			archinstall.log(f" ** You will now select which partitions to use by selecting mount points (inside the installation). **")
-			archinstall.log(f" ** The root would be a simple / and the boot partition /boot (as all paths are relative inside the installation). **")
+			archinstall.log(
+				f" ** You will now select which partitions to use by selecting mount points (inside the installation). **"
+			)
+			archinstall.log(
+				f" ** The root would be a simple / and the boot partition /boot (as all paths are relative inside the installation). **"
+			)
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. Doesn't make sense.